modules/cnv-initiating-vm-migration-cli.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-migrate-vmi.adoc
+
+[id="cnv-initiating-vm-migration-cli_{context}"]
+= Initiating live migration of a virtual machine instance in the CLI
+
+Initiate a live migration of a running virtual machine instance by creating a 
+*VirtualMachineInstanceMigration* object in the cluster and referencing the name
+ of the virtual machine instance.
+
+.Procedure
+
+. Create a *VirtualMachineInstanceMigration* configuration file for the 
+virtual machine instance to migrate. For example, `vmi-migrate.yaml`:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstanceMigration
+metadata:
+  name: migration-job
+spec:
+  vmiName: vmi-fedora
+----
+
+. Create the object in the cluster:
++
+----
+oc create -f vmi-migrate.yaml
+----
+
+The *VirtualMachineInstanceMigration* object triggers a live migration of the 
+virtual machine instance and exists until the migration is complete. 
```

Comment:
As I know, *VirtualMachineInstanceMigration* object will exist all time while VMI exists (it will be removed only after stopping/deleting VM)

Reply:
Yes. Live migration object will remain as long as the VMI object exist or if manually deleted. In fact, there can be multiple migration object for each VMI.

Reply:
Updated to read: "The *VirtualMachineInstanceMigration* object triggers a live migration of the
virtual machine instance. This object exists in the cluster for as long as the
virtual machine instance is running, unless manually deleted."